app/grandchallenge/reader_studies/templates/reader_studies/display_set_add_interface.html

```diff
@@ -0,0 +1,15 @@
+{% load crispy_forms_tags %}
+
+<div class="card-body py-1" id="ds-content-{{ object.id }}">
+  <form action="{% url 'reader-studies:display-set-add-interface' pk=object.pk %}" method="post" class="ds-form" data-hx-get="{% url 'reader-studies:display-set-update' pk=object.pk %}" data-hx-target="#ds-content-{{ object.id }}" data-hx-swap="outerHTML" id="form-{{ object.pk }}">
```

Comment:
Again, not sure if you need the `data-hx-get` attribute here (see the other comment).